pygly/subtreeByWenjin.py

```diff
@@ -0,0 +1,172 @@
+import itertools
```

Comment:
I'd rather leave this for another branch if possible. Can you remove this from the set of commits in this branch?